src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -1294,9 +1299,7 @@
     <Resource Include="UI\Images\icon-whats-new-small.png" />
     <Resource Include="UI\Images\icon-dictionary-small.png" />
   </ItemGroup>
-  <ItemGroup>
-    <Folder Include="ViewModels\Watch3D\shaderSource\" />
```

Comment:
any reason for this change? This just stops the hlsl files from being visible in visual studio right?

Reply:
up to you, files can still be edited if you use the view external files button.

Reply:
Let me bring it back.